lib/webdriveragent.js

```diff
@@ -393,10 +421,15 @@ class WebDriverAgent {
   }
 
   async quit () {
-    this.log.info('Shutting down sub-processes');
-
-    await this.xcodebuild.quit();
-    await this.xcodebuild.reset();
+    if (this.xctestWDA) {
+      this.log.info('Stopping the XCTest session');
+      this.xctestWDA.stop();
```

Comment:
is this method synchronous?

Reply:
no, https://github.com/appium/appium-ios-device/blob/master/lib/xctest.js#L262